packages/server/src/sdk/app/rows/search/sqs.ts

```diff
@@ -106,6 +135,26 @@ function buildTableMap(tables: Table[]) {
   return tableMap
 }
 
+function reverseUserColumnMapping(rows: Row[]) {
+  const prefixLength = USER_COLUMN_PREFIX.length
+  return rows.map(row => {
+    const finalRow: Row = {}
+    for (let key of Object.keys(row)) {
+      // it should be the last prefix
+      const lastIndex = key.lastIndexOf(USER_COLUMN_PREFIX)
+      if (lastIndex !== -1 && lastIndex < key.length - prefixLength) {
+        // cut out the prefix
+        const newKey =
+          key.slice(0, lastIndex) + key.slice(lastIndex + prefixLength)
+        finalRow[newKey] = row[key]
+      } else {
+        finalRow[key] = row[key]
+      }
+    }
+    return finalRow
+  })
+}
```

Comment:
Are we not running the risk here of the user having columns that contain the prefix? e.g. does this not fail for the column name `data_foo`? It gets prefixed to `data_data_foo` and then we find the last occurrence of `data_`, which is not the correct one.

Would be worth writing some tests regardless that create column names that contain the prefix in various ways to make sure we do this wrapping/unwrapping correctly.